tests/safety/test.c

```diff
@@ -63,6 +63,18 @@ void set_timer(int t){
   timer.CNT = t;
 }
 
+void rx(CAN_FIFOMailBox_TypeDef *to_send) {
+  safety_rx_hook(to_send);
+}
+
+int tx(CAN_FIFOMailBox_TypeDef *to_push) {
+  return safety_tx_hook(to_push);
+}
+
+int set_mode(uint16_t  mode, int16_t param) {
+  return safety_set_mode(mode, param);
```

Comment:
simplify `tests/safety` and use this instead of initting each car liek we do now